### PR TITLE
Add health check to depends on for Python services 

### PR DESCRIPTION
**Pull request by jbregstone**
# Feature Name 

### What was the problem?
Python services failed to wait for Rabbit MQ before trying to connect. 

### How does this fix it?
Docker compose has been updated for the Python services to properly wait for Rabbit MQ to be healthy.  

### Jira Tickets

- [MCP-1810](https://amida.atlassian.net/browse/MCP-1810)

## How to test this PR

- Step 1
- Run the gradle commands to start up the project. See in the logs that python services are successful on the first attempt at connection. 
- Step 2 

## Reminders

- Review [Pull-Requests](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests) guidelines
- [ ] If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) and
[Roadmap](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Roadmap) wiki pages
- [ ] If changing software behavior, post a link to the PR in Slack channel #benefits-virtual-regional-office

### CODE REVIEW

File: app/src/docker/docker-compose.yml [hunk 1]
```diff
@@ -141,8 +143,8 @@ services:
       RABBITMQ_PLACEHOLDERS_USERPASSWORD: guest
       REDIS_PLACEHOLDERS_HOST: redis1
     depends_on:
-      - rabbitmq1
-      - redis1
```
Comment (anthonyvelazquez) [inline app/src/docker/docker-compose.yml:145]: Can you just add the `redis1` condition back? Other than that, good to go

**Review by anthonyvelazquez — APPROVED** [review]

File: app/src/docker/docker-compose.yml (hunk 1 above)
Comment (jbregstone) [reply app/src/docker/docker-compose.yml:145]: I was having trouble with the syntax there, it did not like mixing the dashes with update direct reference

**Review by au2 — APPROVED** [review]

**Review by au2 — CHANGES_REQUESTED** [review]
Please add redis check for pdf generation. You can use 
      redis1:
        condition: service_started

File: app/src/docker/docker-compose.yml (hunk 1 above)
Comment (au2) [reply app/src/docker/docker-compose.yml:145]: Do  
    redis1:
        condition: service_started

**Review by anthonyvelazquez — APPROVED** [review]

**Review by au2 — APPROVED** [review]

**Pull request merged**